packages/swapper/src/swappers/thorchain/types.ts

```diff
@@ -21,6 +21,21 @@ export type PoolResponse = {
   volume24h: string
 }
 
+type out = {
+  txID: string
+}
+type action = {
+  date: string
+  height: string
+  out: [out]
+  status: string
+  type: string
+}
+
+export type ActionsResponse = {
+  actions: [action]
```

Comment:
`T[]` (preferred for consistency) or `Array<T>` 🙏🏽

Reply:
it really doesnt matter

Reply:
`[action]` is an array of length exactly 1

Reply:
derp

Reply:
fixed